src/static/js/routes/offices/index.js

```diff
@@ -15,11 +15,9 @@ function init() {
     return;
   }
 
-  var breakpointPx = 599;
-
   $( '.expandable__mobile-only' ).each( function() {
     // ESLint no-new rule ignored
-    new MobileOnlyExpandable( $( this ), breakpointPx ); // eslint-disable-line
+    new MobileOnlyExpandable( $( this ) ); // eslint-disable-line
```

Comment:
If you prefer, I found the rule can be ignored on one line if you just ignore the additional issues that come up:

`// eslint-disable-line no-new, no-inline-comments, max-len`

I kind of like it better because there isn't a need for two comments and it won't be ignoring everything, as it currently is doing.